ambry-api/src/main/java/com.github.ambry/config/SSLConfig.java

```diff
@@ -145,6 +145,20 @@
   @Default("com.github.ambry.commons.JdkSslFactory")
   public final String sslFactory;
 
+  /**
+   * The implementation of {@link com.github.ambry.commons.SSLFactory} to use.
+   */
+  @Config("ssl.http2.factory")
+  @Default("com.github.ambry.rest.NettySslHttp2Factory")
+  public final String sslHttp2Factory;
+
+  /**
+   * Use self signed HTTP2 SSL certification if true.
```

Comment:
Could you add a disclaimer to the javadoc to mention that production systems should not use this option as it is relatively insecure.